### ISSUE TITLE
inception_resnet_v2 for transfer learning

### ISSUE THREAD

**Issue by GlastonburyC** [tensorflow/models#730]
I am using the following script to load inception_resnet_v2 and its checkpoint.

```
import tensorflow as tf
slim = tf.contrib.slim
from PIL import Image
from inception_resnet_v2 import *
import numpy as np

checkpoint_file = 'inception_resnet_v2_2016_08_30.ckpt'

input_tensor = tf.placeholder(tf.float32, shape=(None,299,299,3), name='input_image')
scaled_input_tensor = tf.scalar_mul((1.0/255), input_tensor)
scaled_input_tensor = tf.sub(scaled_input_tensor, 0.5)
scaled_input_tensor = tf.mul(scaled_input_tensor, 2.0)

# has to be scaled.
#im = 2*(im/255.0)-1.0

#Load the model
sess = tf.Session()
arg_scope = inception_resnet_v2_arg_scope()
with slim.arg_scope(arg_scope):
  logits, end_points = inception_resnet_v2(scaled_input_tensor, is_training=True,num_classes=8,reuse=True,scope='InceptionResnetV2')
```
How would I go about removing the last layer, adding a 2DPooling and dense layer so I can classify on another problem with only 8 labels?

Thank you.

**Comment by michaelisard** [comment]
This kind of usage question is better asked on [stackoverflow](http://stackoverflow.com/questions/tagged/tensorflow)
